sycl-jit/jit-compiler/lib/KernelFusion.cpp

```diff
@@ -235,6 +236,31 @@ extern "C" JITResult fuseKernels(View<SYCLKernelInfo> KernelInformation,
   return JITResult{FusedKernelInfo};
 }
 
+extern "C" JITResult compileSYCL(const char *SYCLSource,
+                                 View<IncludePair> IncludePairs,
+                                 View<const char *> UserArgs,
+                                 const char *DPCPPRoot) {
```

Comment:
DPCPPRoot  is not like the other arguments.  Doesn't it seem like the routine should be able to figure that out itself, rather than being provided?

Reply:
Makes sense; it's detected from inside the JIT library now.